### ISSUE TITLE
postgresql 10.0-1: `postgresql-check-db-dir` points to an old PGMAJORVERSION.

### ISSUE THREAD

**Issue by misaka4e21** [AOSC-Dev/aosc-os-abbs#864]
In file ` aosc-os-abbs/extra-database/postgresql/autobuild/overrides/usr/bin/postgresql-check-db-dir`:
```bash
# PGMAJORVERSION is major version
PGMAJORVERSION=9.5
# PREVMAJORVERSION is the previous major version, e.g., 8.4, for upgrades
PREVMAJORVERSION=9.4
```

The outdated `PGMAJORVERSION` which differs from the current `PG_VERSION` `10` prevents `postgresqld.service` from launch:
```
  Process: 9825 ExecStart=/usr/bin/pg_ctl -s -D ${PGROOT}/data start -w -t 120 (code=exited, status=1/FAILURE)
  Process: 10108 ExecStartPre=/usr/bin/postgresql-check-db-dir ${PGROOT}/data (code=exited, status=1/FAILURE)

11月 19 12:33:09 aosc systemd[1]: Starting PostgreSQL database server...
11月 19 12:33:09 aosc postgres[10108]: An old version of the database format was found.
11月 19 12:33:09 aosc postgres[10108]: See https://wiki.archlinux.org/index.php/PostgreSQL#Upgrading_PostgreSQL
11月 19 12:33:09 aosc systemd[1]: postgresqld.service: Control process exited, code=exited status=1
11月 19 12:33:09 aosc systemd[1]: postgresqld.service: Failed with result 'exit-code'.
11月 19 12:33:09 aosc systemd[1]: Failed to start PostgreSQL database server.
```

Manually edited with a `PGMAJORVERSION=10`, the service started normally.

**Comment by MingcongBai** [comment]
Fixed with https://github.com/AOSC-Dev/aosc-os-abbs/commit/49438f29a56df77d5951ae4bc53f12a4ed78c571. Closing.

**Comment by cthbleachbit** [comment]
So this has happened again as of `9850611415c7f1a7d33a547374065b600b5e0947`.

**Comment by MingcongBai** [comment]
Addressed again with bcb161bf8c53c2b1dc7195574f780e56e33f7196. Closing.

Please open new issues if this stupid issue arises again (and again, we should work on a file template mechanism for Autobuild3).
